src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
     public int getUserScore(String[] records, String email) {
-        return 0;
+
+        for (String record : records) {
+            if (record.substring(0,record.indexOf(":")).equals(email)) {
+                return Integer.parseInt(record.substring(record.indexOf(":") + 1,
```

Comment:
that's too complicated: 
String[] splitedRecord = record.split(":");
if (splitedRecord[0].equals(email); {
return Integer.parseInt(splitedRecord[1]);